frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/BaseSlideOutBranchAction.java

```diff
@@ -54,19 +49,18 @@ public void doSlideOut(AnActionEvent anActionEvent, BaseGitMacheteNonRootBranch
     String branchName = branchToSlideOut.getName();
     Project project = ActionUtils.getProject(anActionEvent);
     var branchLayout = ActionUtils.getBranchLayout(anActionEvent);
-    var gitMacheteFilePath = ActionUtils.getGitMacheteFilePath(anActionEvent);
-    if (branchLayout.isEmpty() || gitMacheteFilePath.isEmpty()) {
-      LOG.warn("Skipping the action because branchLayout and/or gitMacheteFilePath is empty");
+    var branchLayoutWriter = ActionUtils.getBranchLayoutWriter(anActionEvent);
+    if (branchLayout.isEmpty() || branchLayoutWriter.isEmpty()) {
+      LOG.warn("Skipping the action because branch layout and/or branch layout writer is undefined");
```

Comment:
I'm gonna rephrase this everywhere on my own PR

Reply:
:this-is-fine: